spec/function-builder.spec.ts

```diff
@@ -165,21 +165,21 @@ describe('FunctionBuilder', () => {
 
   it('should throw an error if user chooses an invalid region', () => {
     expect(() => {
-      return functions.region('unsupported');
+      return functions.region('unsupported' as any);
     }).to.throw(Error, 'region');
 
     expect(() => {
-      return functions.region('unsupported').runWith({
+      return functions.region('unsupported' as any).runWith({
         timeoutSeconds: 500,
       } as any);
     }).to.throw(Error, 'region');
 
     expect(() => {
-      return functions.region('unsupported', 'us-east1');
+      return functions.region('unsupported' as any, 'us-east1');
     }).to.throw(Error, 'region');
 
     expect(() => {
-      return functions.region('unsupported', 'us-east1').runWith({
+      return functions.region('unsupported' as any, 'us-east1').runWith({
```

Comment:
All of these can be changed to avoid `any`, I can do that if that's your preference. `any` is much simpler here.

```suggestion
      return functions
        .region(
          ('unsupported' as unknown) as Parameters<
            typeof functions.region
          >[number],
          'us-east1'
        )
        .runWith({
```

Reply:
I feel it looks too verbose with using `unknown`, let's keep using `any`. Thanks!